### DISCUSSION TITLE
All radio inputs across the GH UI have `opacity: 0`

### DISCUSSION THREAD

**Discussion by julik** [community/community#6852] [Other Feature Feedback, Questions, & Ideas] [answered]
Starting today, in a recent version of Chrome, all "radio" inputs - such as the ones in the PR review popover - are hidden. When examining stylesheets I noticed that the `input` is set to `opacity: 0` via an "injected stylesheet". So far I have also seen this in the repository visibility switch modal - again, the inputs are supposed to be radios, but they are invisible.

Likely a UI regression somewhere. Observing since this morning (so for about 15 hours now).

**Comment by rm3l** [comment]
Hi there !

Similar issue here: when attempting to create a new repo, I couldn't select the repository visibility, because Public vs Private checkboxes are hidden.

![image](https://user-images.githubusercontent.com/593208/139221328-634a37ba-f9c8-4969-bc4a-5bcc7070c21c.png)

I am using Chrome `Version 95.0.4638.54 (Official Build) (64-bit)` on `Debian 11`

NOTE: I can see the checkboxes on Firefox (`78.15.0esr (64-bit)`)

**Comment by julik** [comment]
https://github.com/github/feedback/discussions/6840

**Comment by gaknoll** [comment, marked as answer]
Thanks @julik & @rm3l for reporting this - I've logged the bug and raised the issue with the team 🙏🏽

**Reply by gaknoll** [reply]
@julik @rm3l After digging into this, it looks like it's caused by the "Save to Pocket" Chrome extension. Pocket is aware and we hope they get this resolved quickly!
https://twitter.com/Phacks/status/1453343546371674118

**Reply by rm3l** [reply]
> @julik @rm3l After digging into this, it looks like it's caused by the "Save to Pocket" Chrome extension. Pocket is aware and we hope they get this resolved quickly! https://twitter.com/Phacks/status/1453343546371674118

Thanks @gaknoll for the info. Indeed, I updated the "Save to Pocket" Chrome extension, and the issue is fixed now.
